plugins/woocommerce/tests/performance/requests/merchant/add-order.js

```diff
@@ -0,0 +1,590 @@
+/* eslint-disable no-shadow */
+/* eslint-disable import/no-unresolved */
+/**
+ * External dependencies
+ */
+import { sleep, check, group } from 'k6';
+import http from 'k6/http';
+import {
+	randomIntBetween,
+	findBetween,
+} from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+	base_url,
+	cot_status,
+	addresses_guest_billing_first_name,
+	addresses_guest_billing_last_name,
+	addresses_guest_billing_company,
+	addresses_guest_billing_country,
+	addresses_guest_billing_address_1,
+	addresses_guest_billing_address_2,
+	addresses_guest_billing_city,
+	addresses_guest_billing_state,
+	addresses_guest_billing_postcode,
+	addresses_guest_billing_phone,
+	addresses_guest_billing_email,
+	payment_method,
+	think_time_min,
+	think_time_max,
+} from '../../config.js';
+import {
+	htmlRequestHeader,
+	jsonAPIRequestHeader,
+	jsonRequestHeader,
+	commonRequestHeaders,
+	commonGetRequestHeaders,
+	contentTypeRequestHeader,
+	commonPostRequestHeaders,
+	commonAPIGetRequestHeaders,
+	commonNonStandardHeaders,
+} from '../../headers.js';
+
+// Change URL if COT is enabled and being used
+let admin_new_order_base;
+let admin_open_order_base;
+let admin_update_order_base;
+let admin_update_order_id;
+let admin_update_order_params;
+
+if ( cot_status === true ) {
+	admin_new_order_base = 'admin.php?page=wc-orders&action=new';
+	admin_update_order_base = 'admin.php?page=wc-orders&action=edit';
+} else {
+	admin_new_order_base = 'post-new.php?post_type=shop_order';
+	admin_update_order_base = 'post.php';
+}
+
+const date = new Date();
+const order_date = date.toJSON().slice( 0, 10 );
```

Comment:
Is there a simpler way to extract the date in the desired format? Perhaps using a dedicated Date function or something?

Reply:
This is done this to match the date format of the request.